ci_framework/roles/edpm_build_images/README.md

```diff
@@ -0,0 +1,8 @@
+## Role: build_images
+Please explain the role purpose.
+
+### Privilege escalation
+If apply, please explain the privilege escalation done in this role.
+
```

Comment:
TODO
- Add support for passing yum repos and distro images.

Reply:
we have added this support